hack/generator/pkg/codegen/golden_files_test.go

```diff
@@ -146,41 +146,42 @@ func NewTestCodeGenerator(testName string, path string, t *testing.T, testConfig
 	switch genPipeline {
 	case config.GenerationPipelineAzure:
 		codegen.RemoveStages(
-			"deleteGenerated",
-			"rogueCheck",
-			"createStorageTypes",
-			"injectOriginalGVKFunction",
-			"injectOriginalVersionFunction",
-			"injectOriginalVersionProperty",
-			"injectPropertyAssignmentFunctions",
-			//pipeline.InjectHubFunctionStageId,
-			"reportTypesAndVersions")
+			pipeline.DeleteGeneratedCodeStageID,
+			pipeline.CheckForAnyTypeStageID,
+			pipeline.CreateStorageTypesStageID,
+			pipeline.InjectOriginalGVKFunctionStageID,
+			pipeline.InjectOriginalVersionFunctionStageID,
+			pipeline.InjectOriginalVersionPropertyStageID,
+			pipeline.InjectPropertyAssignmentFunctionsStageID,
+			//pipeline.InjectHubFunctionStageID,
```

Comment:
minor: Is this commented out for a reason? I think yes and it will be enabled later? Might be worth a TODO here

Reply:
I don't think we'll want to include this stage in tests, but we can't remove it until it's formally included in the pipeline. Comment added.